app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map(function() {
     });
   });
   this.route('exit');
+  this.route('complete');
```

Comment:
Good question! Does it matter that a user is only supposed to see this route if they log in and have already completed the study? Meaning they will never go from `https://google.com/survey/ --> https://google.com/survey/complete`